proto/src/determined/user/v1/user.proto

```diff
@@ -20,6 +20,14 @@ message User {
   bool active = 4;
   // The user and group on the agent host machine.
   AgentUserGroup agent_user_group = 5;
+  string display_name = 6;
```

Comment:
For `proto` to generate complete docs: this, PatchUser, and the PatchUser display_name should each have a comment above them (starting with // and ending with a period)